fastlane/actions/gitlab_create_project.rb

```diff
@@ -10,13 +10,17 @@ def self.run(params)
         source = Gitlab.client(endpoint: params[:endpoint_src], private_token: params[:api_token_src])
         destination = Gitlab.client(endpoint: params[:endpoint_dst], private_token: params[:api_token_dst])
         original_project = params[:project]
+        if original_project.owner
```

Comment:
Reading this issue [https://gitlab.com/gitlab-com/support-forum/issues/261] I think that makes sense. A project belonging to a group has no owner. It belongs to the group. 
(communist approach ;-) )